src/Hedgehog/Extras/Test/File.hs

```diff
@@ -336,3 +337,9 @@ appendFileTimeDelta filePath offsetTime = GHC.withFrozenCallStack $ do
   baseTime <- H.noteShowIO DTC.getCurrentTime
   let delay = DTC.diffUTCTime baseTime offsetTime
   appendFile filePath $ show @DTC.NominalDiffTime delay <> "\n"
+
+-- | Asserts that the given directory is missing.
+assertDirectoryMissing :: (MonadTest m, MonadIO m, HasCallStack) => FilePath -> m ()
```

Comment:
Can you add a function for the opposite case, where directory exists, too?

Reply:
Good idea :+1: